src/types/enums.ts

```diff
@@ -15,3 +15,232 @@ export type CompareResultsTableHeaders =
   | 'Total Runs';
 
 export type ConfidenceText = 'high' | 'med' | 'low';
```

Comment:
[suggestion] I see you use med here but not updated the mock data to use medium.

Reply:
will need to confirm from Beatrice if it's `med` or `medium`

Reply:
👌🏻 I think in perfherder it shows 'med' but yes we need to confirm this.